packages/components/src/components/domainproperties/list/models.spec.ts

```diff
@@ -1,10 +1,51 @@
 import {ListModel} from "./models";
-
+import {DEFAULT_LIST_SETTINGS} from "../../../test/data/constants";
+import getDomainDetailsJSON from "../../../test/data/property-getDomainDetails.json";
 
 describe('ListModel', () => {
-    test("", () => {
-        const model = ListModel.create({
 
-        })
+    test("new model", () => {
+        const newModel = ListModel.create(null, DEFAULT_LIST_SETTINGS);
+
+        expect(newModel.getIn(['domain', 'name'])).toEqual(undefined);
+        expect(newModel.getIn(['domain', 'domainId'])).toEqual(null);
+        expect(newModel.getIn(['name'])).toEqual(null);
+        expect(newModel.getIn(['keyType'])).toEqual(null);
+
+        expect(newModel.isNew()).toEqual(true);
+    });
+
+    test("existing model", () => {
+        const existingModel = ListModel.create(getDomainDetailsJSON);
+
+        expect(existingModel.getIn(['domain', 'name'])).toEqual('NIMHDemographics');
+        expect(existingModel.getIn(['domain', 'domainId'])).toEqual(2280);
+        expect(existingModel.getIn(['name'])).toEqual('NIMHDemographics');
+        expect(existingModel.getIn(['keyType'])).toEqual('Integer');
+        expect(existingModel.getIn(['keyName'])).toEqual('SubjectID');
+
+        expect(existingModel.isNew()).toEqual(false);
```

Comment:
same comment as above but for the false case, try
`expect(newModel.isNew()).toBeFalsy();`